### DISCUSSION TITLE
Implement A Private Filter List In UBO

### DISCUSSION THREAD

**Discussion by BlazeFTL** [uBlockOrigin/uBlock-issues#2990] [Ideas] [answered]
This Would Be Great For Site Owners Who Like To ByPass Or Break The Site Actively...They Also Do This By Looking At The Filters We Update

So A Filter List Which Is Private Only Accessable By UBO Maintainers For These Type Of Site Who Update Anti Adb Regularly..Not Visible On Logger Either

The Idea Is To Not Let The Site Owner See How We Are Bypassing The Anti Adb
It Would Definitely Increase If Not Stop The Amount Of Time A Filter Works Before Site Owner Steps In

**Comment by peace2000** [comment]
Discussed internally once https://github.com/uBlockOrigin/uBlock-discussions/discussions/776. Skilled admins could still figure out what uBO is doing.

**Reply by BlazeFTL** [reply]
Yeah Currently For Most Sites We Are Using rmnt & rpnt Which Would Be Difficult To Trace...

**Reply by iam-py-test** [reply]
uBo still has to load the filterlist, so someone could just inspect the network requests made by uBo to find the list, right?
Thanks

**Comment by gorhill** [comment, marked as answer]
I am just philosophically against doing "secret" stuff with regard to uBO. I want everything to be developed in the open, whether code or filter lists. Beside, in the end all ends up to be public, there is no point anyway regardless of philosophy.
